editor.planx.uk/src/api/upload.ts

```diff
@@ -40,14 +43,19 @@ function handleUpload(
   formData.append("file", file);
   formData.append("filename", file.name);
 
+  const passport = useStore.getState().computePassport();
+
+  // Private uploads for test applicaitons should be handled by the staging environment
   const paths = {
-    public: "public-file-upload",
-    private: "private-file-upload",
+    public: `${process.env.REACT_APP_API_URL}/public-file-upload`,
+    private: useStagingUrlIfTestApplication(passport)(
+      `${process.env.REACT_APP_API_URL}/private-file-upload`
```

Comment:
This might be over cautious, but it's possible for an Editor to have a passport with "test test" whilst building a flow and uploading public files. In these cases, we don't want those files to be inaccessible from the prod public interface.